packages/static-store/README.md

```diff
@@ -0,0 +1,112 @@
+<p>
+  <img width="100%" src="https://assets.solidjs.com/banner?type=Primitives&background=tiles&project=static-store" alt="Solid Primitives Static Store">
+</p>
+
+# @solid-primitives/static-store
+
+[![turborepo](https://img.shields.io/badge/built%20with-turborepo-cc00ff.svg?style=for-the-badge&logo=turborepo)](https://turborepo.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/static-store?style=for-the-badge&label=size)](https://bundlephobia.com/package/@solid-primitives/static-store)
+[![version](https://img.shields.io/npm/v/@solid-primitives/static-store?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/static-store)
+[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsolidjs-community%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-2.json)](https://github.com/solidjs-community/solid-primitives#contribution-process)
+
+Primitives for creating small reactive objects that doesn't change their shape over time - don't need a proxy wrapper.
+
+- [`createStaticStore`](#createStaticStore) - Creates a writable static store object.
+- [`createDerivedStaticStore`](#createDerivedStaticStore) - Creates a static store that is derived from a source function.
+
+## Installation
+
+```bash
+npm install @solid-primitives/static-store
+# or
+yarn add @solid-primitives/static-store
+# or
+pnpm add @solid-primitives/static-store
+```
+
+## `createStaticStore`
+
+A shallowly wrapped reactive store object. It behaves similarly to the createStore, but with limited features to keep it simple. Designed to be used for reactive objects with static keys, but dynamic values, like reactive Event State, location, etc.
```

Comment:
I find this description slightly confusing. Maybe rephrase it like:

"Creates read-only object that is solely reactive on its first level and for the enumerable properties specified in the initial value, and a setter. It behaves similarly to createStore, but with limited features to keep it simple and performant. Designed to be used for reactive objects with static keys and dynamic values, like reactive Event State, location, etc."

Reply:
hmm I love that, thanks 👍